bigbluebutton-html5/imports/ui/core/utils/keyboardRove.ts

```
@@ -0,0 +1,51 @@
+    return;
+  }
+
+  if (ev.code === 'ArrowDown' || ev.code === 'ArrowUp') {
+    ev.preventDefault();
+    ev.stopPropagation();
+    const sum = ev.code === 'ArrowDown' ? 1 : -1;
+    const el = selectedRef.current;
+    if (el) {
+      try {
+        const currentId = Number.parseInt(el.id.split('-')[2], 10);
+        if (isNaN(currentId)) throw new Error('Invalid ID format');
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)